### DISCUSSION TITLE
Upgrade component in Fedora Silverblue

### DISCUSSION THREAD

**Discussion by v1gnesh** [evilsocket/opensnitch#1061] [Q&A] [answered]
When I initially installed (v1.6.2), I did → Right click, open with Software Install.
If I try the same now (with v1.6.3), it only shows an uninstall option.

The linked wiki says an uninstall is not necessary.
I hope that’s not necessary in Silverblue… ?

I think this is best installed as an OS layer (as is the concept with Silverblue... not within toolbox or as a flatpak app), as this is a network activity monitor.

Any guidance?
If needed, is it safe to uninstall 1.6.2 UI and install 1.6.3, without losing settings, etc.?

**Comment by gustavo-iniguez-goya** [comment, marked as answer]
hey @v1gnesh ,

woah, I really thought I answered this question, sorry. 

Yes, it's safe to uninstall 1.6.2 and install 1.6.3. UI Settings are saved under ~/.config/opensnitch/, and that directory is not removed on uninstall.

**Reply by v1gnesh** [reply]
Thank you!

**Reply by tuxPT** [reply]
I add that in rpm-ostree distros, in order to do upgrades of locally installed rpms(through rpm files) the equivalent command to upgrade/reinstall/downgrade is:

```
rpm-ostree install package.rpm --uninstall package
```
